src/dataset/dataset.service.js

```diff
@@ -207,4 +208,5 @@ angular.module('vlui')
   };
 
   return Dataset;
-});
+}
+]);
```

Comment:
The indentation of this file is weird—I'll fix that in a follow-up PR once this one's in